src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,41 @@
+.menu {
+  overflow: scroll;
```

Comment:
Replace 'overflow: scroll;' with 'overflow-y: auto;' to show scrollbar only when necessary and only on the vertical axis.